internal_workflow.go

```diff
@@ -410,33 +411,26 @@ func getDispatcher(ctx Context) dispatcher {
 // executeDispatcher executed coroutines in the calling thread and calls workflow completion callbacks
 // if root workflow function returned
 func executeDispatcher(ctx Context, dispatcher dispatcher) {
-	checkUnhandledSigFn := func(ctx Context) {
-		us := getWorkflowEnvOptions(ctx).getUnhandledSignals()
-		if len(us) > 0 {
-			getWorkflowEnvironment(ctx).GetLogger().Warn("Workflow has unhandled signals",
-				zap.Strings("SignalNames", us))
-			// TODO: We don't have a metrics added to workflow environment yet,
-			// this need to be reported as a metric.
-		}
-	}
-
+	env := getWorkflowEnvironment(ctx)
 	panicErr := dispatcher.ExecuteUntilAllBlocked()
 	if panicErr != nil {
-		env := getWorkflowEnvironment(ctx)
-		env.GetLogger().Error("Dispatcher panic.",
-			zap.String("PanicError", panicErr.Error()),
-			zap.String("PanicStack", panicErr.StackTrace()))
-		checkUnhandledSigFn(ctx)
```

Comment:
They are moved to task handler layer. Because the env.GetLogger() will not log in replay, but we want to log anyway for all panic.